src/main/resources/assets/appliedenergistics2/lang/en_US.lang

```diff
@@ -403,9 +403,9 @@ gui.tooltips.appliedenergistics2.SearchMode=Search Box Mode
 gui.tooltips.appliedenergistics2.PartitionStorageHint=Configures Partition based on currently stored items.
 gui.tooltips.appliedenergistics2.ClearSettings=Clear Config/Settings
 gui.tooltips.appliedenergistics2.Craftable=Craftable
-gui.tooltips.appliedenergistics2.MoveWhenEmpty=Move to output when empty.
+gui.tooltips.appliedenergistics2.MoveWhenEmpty=Move to output when cell is empty or network was emptied.
 gui.tooltips.appliedenergistics2.MoveWhenWorkIsDone=Move to output when work is done.
-gui.tooltips.appliedenergistics2.MoveWhenFull=Move to output when full.
+gui.tooltips.appliedenergistics2.MoveWhenFull=Move to output when cell is full or network was filled.
```

Comment:
As a user, I have no idea what "network emptied" or "network filled" mean.

The tooltip should be better

Reply:
Suggestions welcome. "Network emptied" and "Network filled" means just that, the network the IO port is attached to becomes empty or becomes completely full.

For a 100% correct explanation of the behavior, the tooltip would probably need to be 2-3 sentences, so maybe it needs to be moved somewhere else, like the IO port tooltip itself.